### DISCUSSION TITLE
Manylinux 2010

### DISCUSSION THREAD

**Discussion by renefritze** [pymor/pymor#1435] [Q&A] [answered]
Is there anybody out there who needs manylinux 2010 wheels? Some packages we (indirectly) depend on are no longer releasing 2010 compatible packages which in turn makes our 2010 wheels harder to build and install. 

If nobody can come up with a very good reason to keep 2010 till the end of this, I shall drop 2010 from our stack.

**Comment by sdrave** [comment]
Can you say, under which conditions manylinux 2010 might be required?

**Reply by renefritze** [reply]
Yes, if you're running a linux platform with glibc 2.12.

**Comment by renefritze** [comment, marked as answer]
Actually, I straight up forgot we're pure python since the last release and already only ship a `py3-none-any.whl`, so I can drop the whole manylinux stack.

**Reply by renefritze** [reply]
[I very much like this PR](https://github.com/pymor/docker/pull/93)
